disorder-jack/src/Disorder/Jack/Property.hs

```diff
@@ -46,23 +47,18 @@ module Disorder.Jack.Property (
   , forAllProperties
   ) where
 
-import           Data.Eq (Eq(..))
 import           Data.Foldable (for_, traverse_)
-import           Data.Function (($), (.))
-import           Data.Functor (Functor(..))
 import qualified Data.List as List
 import           Data.Monoid ((<>))
-import           Data.String (String)
 import           Data.Text (Text)
 import qualified Data.Text as T
 
 import           Disorder.Jack.Core
 import           Disorder.Jack.Tree
 
-import           System.IO (IO, putStrLn)
-
-import           Text.Show (Show)
 import           Text.Show.Pretty (ppShow)
+import qualified Text.Show.Pretty as Pretty
+import           Prelude
```

Comment:
-1 for this, it's hard to get code to compile across GHC version without `NoImplicitPrelude` (usually it requires CPP)